examples/qtmegachatapi/callListener.cpp

```diff
@@ -0,0 +1,39 @@
+#include "callListener.h"
+
+CallListener::CallListener(MegaChatApi *megaChatApi, CallGui *callGui)
+ : MegaChatVideoListener()
+{    
+    mMegaChatApi = megaChatApi;
+    mCallGui = callGui;
+    megaChatVideoListenerDelegate = new QTMegaChatVideoListener(mMegaChatApi, this);    
+}
+ CallListener::~ CallListener()
+ {
+     delete megaChatVideoListenerDelegate;     
+ }
+
+ QImage *CallListener::CreateFrame(int width, int height, char *buffer, size_t size)
+ {
+     if((width == 0) || (height == 0))
+     {
+         return NULL;
+     }
+
+     char *copyBuff = new char[size];
+     memcpy(copyBuff,buffer,size);
+     unsigned char *auxBuf = reinterpret_cast<unsigned char*> (copyBuff);
```

Comment:
I think you can directly define `copyBuff` as `unsigned char*`. `memcpy()` won't complain (or you can use casting there) :)